contrib/i3status-rust/template.py

```diff
@@ -1,13 +1,18 @@
 pkgname = "i3status-rust"
 pkgver = "0.33.0"
-pkgrel = 0
+pkgrel = 1
 build_style = "cargo"
-make_build_args = ["--features=notmuch"]
+make_build_args = [
+    "--no-default-features",
+    "--features=notmuch,pipewire,pulseaudio,icu_calendar",
```

Comment:
why is pulseaudio here if you removed libpulse-devel

Reply:
it's also missing maildir

Reply:
enabling libpulse is the default, when disabled it breaks the audio source block so I chose to keep it

Reply:
as for the maildir I just didn't see anything in the ports tree that could provide it so it seemed superfluous.

Reply:
you should keep libpulse-devel too then  

> as for the maildir I just didn't see anything in the ports tree that could provide it  

mbsync is one, some others can expose one too iirc

Reply:
ok.